atlasdb-api/src/main/java/com/palantir/atlasdb/transaction/api/OpenTransactions.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * (c) Copyright 2020 Palantir Technologies Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.palantir.atlasdb.transaction.api;
+
+import java.util.List;
+import java.util.Optional;
+
+import com.palantir.lock.watch.IdentifiedVersion;
+import com.palantir.lock.watch.TransactionsLockWatchEvents;
+
+public interface OpenTransactions {
+
+    List<OpenTransaction> getTransactions();
+
+    /**
+     * Returns a condensed view of new lock watch events since lastKnownVersion for the set of started transactions.
+     *
+     * @param lastKnownVersion exclusive start version to get events from
+     */
+    TransactionsLockWatchEvents getEvents(Optional<IdentifiedVersion> lastKnownVersion);
```

Comment:
Should we document/put in the contract the behaviour around FAILUREs at -1 (for systems with no-op managers i.e. embedded)? I don't remember exactly how we handle that case

Reply:
Discussed offline: this ends up just sending a dummy SNAPSHOT always, meaning lock watches are effectively off.